src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -695,4 +707,45 @@ private function getTemplateRegistry(string $adminCode): TemplateRegistryInterfa
 
         throw new ServiceNotFoundException($serviceId);
     }
+
+    /**
+     * @param object|array $listElement
+     *
+     * @return array array containing object and value
```

Comment:
```suggestion
     * @return array containing object and value
```

Reply:
Is that right? If docs are generated the description of the parameter will be 'containing object and value' because array is interpreted as the type.

Reply:
You're right. Maybe it can be improved this way then
`@return array An array with ...`

I found somewhere else in this project 
```
* @return array An array of entities
```

Reply:
Thanks